packages/rest-crud/src/__tests__/acceptance/default-model-crud-rest.acceptance.ts

```diff
@@ -94,6 +97,25 @@ describe('CrudRestController for a simple Product model', () => {
         ],
       });
     });
+
+    it('rejects request with value provided for readOnly property', async () => {
```

Comment:
If we reject readonly properties from post API to, then how will we be able to set it in first place ?

Reply:
This is something that I'd like to discuss. 

What I had in my mind while working on the feature was the generated expression from MySQL. These are generated columns. The values are being set while a new record is inserted. 

We have `generated` but the discoverer marks only the autoincrement properties as generated and it doesn't work with other properties.